squarelet/templates/core/component/services.html

```diff
@@ -52,20 +52,20 @@
         </div>
       </a>
     </li>
-    {% comment %}
     <li>
-      <div class="_cls-serviceIcon">
-        <img src="{% static 'assets/quackbot.svg' %}">
-      </div>
-      <div class="_cls-serviceInfo">
-        <div class="_cls-serviceName">
-          {% trans 'Quackbot' %}
+      <a class="_cls-nostyle" href="{{ settings.BIGLOCALNEWS_URL }}">
+        <div class="_cls-serviceIcon">
+          <img src="{% static 'assets/biglocalnews.svg' %}">
         </div>
-        <div class="_cls-serviceDescription">
-          {% blocktrans %}A task-oriented chat bot for journalists using Slack{% endblocktrans %}
+        <div class="_cls-serviceInfo">
+          <div class="_cls-serviceName">
+            {% trans 'Big Local News' %}
+          </div>
+          <div class="_cls-serviceDescription">
+            {% blocktrans %}Big Local News description here.{% endblocktrans %}
```

Comment:
@palewire Need to insert a short description here

Reply:
The standard Big Local News tagline is:

> Data, tools and collaborations that empower journalists to better cover their community. A project at Stanford University.

If we're looking to save space, we could go with:

> Data, tools and collaborations that empower journalists to better cover their community.

That work for you?

Reply:
Perfect